.prettierrc.yaml

```diff
@@ -0,0 +1,24 @@
+### General
+arrowParens: always
+tabWidth: 4
+useTabs: false
+requirePragma: false
+insertPragma: false
+
+### VueJS
+vueIndentScriptAndStyle: true
+
+### PHP
+phpVersion: "7.4"
```

Comment:
Considering the project starts with 8.0, shouldn't this be using that version at minimum?